perceval/backends/__init__.py

```diff
@@ -27,7 +27,7 @@
 from .jira import Jira, JiraCommand
 from .mbox import MBox, MBoxCommand
 from .stackexchange import StackExchange, StackExchangeCommand
-
+from .jenkins import Jenkins, JenkinsCommand
```

Comment:
Would you like to place this library ordered by name. The same with the keywords on both dictionaries below.